### DISCUSSION TITLE
Link object to a 3rd party API

### DISCUSSION THREAD

**Discussion by elvis0920** [hubs-creators/awesome-mozilla-hubs#8] [Q&A]
Anyone know how we could link an object/avatar in HC to a 3rd part API and where to find in the backend?

**Comment by markusTraber** [comment]
Hi!
This repostory here is to assemble awesome resource surrounding Mozilla Hubs :) Therefore I would recommend you to ask for help at the [official repository](https://github.com/mozilla/hubs) or in the [discord community](https://discord.gg/dFJncWwHun).
